### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 # msftdynamicsgpdocs
-Welcome to the repository for documentation and guides for Microsoft Dynamics GP! The content is available as markdown files (.md), where each file represents an article in the help that publishes to [https://docs.microsoft.com/en-us/dynamics-gp/](https://docs.microsoft.com/en-us/dynamics-gp/). This repo also provides a way for you to actively contribute to the Dynamics GP content, and we welcome your contributions.
+
+Welcome to the repository for documentation and guides for Microsoft Dynamics GP! The content is available as markdown files (.md), where each file represents an article in the help that publishes to [Dynamics GP documentation](https://docs.microsoft.com//dynamics-gp/). This repo also provides a way for you to actively contribute to the Dynamics GP content, and we welcome your contributions.
```
Comment (JustPies) [inline README.md:8]: Potential issue with the revised link.  It should be:
https://docs.microsoft.com/dynamics-gp/

**Comment by JustPies** [comment]
#docutune-wait

File: README.md (hunk 1 above)
Comment (alexbuckgit) [reply README.md]: Good eye :-)

Fixed.

**Comment by alexbuckgit** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@edupont04

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Review by edupont04 — APPROVED** [review]

**Pull request merged**